src/main/java/org/prebid/server/bidder/smaato/SmaatoBidder.java

```diff
@@ -166,6 +168,16 @@ private Site modifySite(Site site, String firstPublisherId) {
         return siteBuilder.build();
     }
 
+    private App modifyApp(App app, String publishedId) {
+        if (app == null) {
+            return null;
+        }
+
+        return app.toBuilder()
```

Comment:
We need unit tests for this case, also can be updated integration test

Reply:
Done